src/pages/bajinaru_ion_simion/AddNote.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import NoteList from "../../components/NoteList"; 
+
+function AddNote() {
+  const [notes, setNotes] = useState([]);
+  const [title, setTitle] = useState("");
+  const [content, setContent] = useState("");
+
+  const handleTitleChange = (e) => {
+    setTitle(e.target.value);
+  };
+
+  const handleContentChange = (e) => {
+    setContent(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+        // your code goes here
+    e.preventDefault();
+    if (title && content) {
+      setNotes([{ title, content }, ...notes]);
```

Comment:
Era mai ok daca separai logica respectiva. (Separation of concerns). 
Astfel codul devine mai usor de mentinut si izoleaza componentele -> `AddNote` - afisarea unei notite.
si o alta componenta => `AddNotes` care va face logica de adaugare.
``` js
import React, { useState } from "react";
import AddNote from "./AddNote.js";
import NoteList from "./NoteList.js";
import "./styles.css";

function AddNotes() {
  const [notes, setNotes] = useState([]);

  const addNote = (title, content) => {
    const newNote = { title, content };
    setNotes((prevNotes) => [...prevNotes, newNote]);
  };

  return (
    <div className="add-notes">
      <AddNote addNote={addNote} />
      <NoteList notes={notes} />
    </div>
  );
}

export default AddNotes;
```